chainer_setup_build.py

```diff
@@ -261,12 +261,26 @@ def get_cython_pkg():
     return pkg_resources.get_distribution('cython')
 
 
+def run_command(cmd):
+    try:
+        subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+    except subprocess.CalledProcessError as e:
+        msg = '''Command %r failed:
+
+  command: %s
+  return code: %d
+  output:
+
+%s''' % (cmd[0], ' '.join(e.cmd), e.returncode, e.output)
```

Comment:
I see. `stderr` seems to be grasped in `check_output`.
https://github.com/python/cpython/blob/2.7/Lib/subprocess.py#L568

Do you think it is good idea to show `stderr` only when it is available?